models/benchmarkable.py

```diff
@@ -298,7 +298,8 @@ def runs_with_high_regressions(
             results = [
                 r
                 for r in results
-                if r["contender"]["language"] in benchmark_langs_filter
+                if r["contender"] is not None and  # see https://github.com/voltrondata-labs/arrow-benchmarks-ci/issues/117
```

Comment:
Yes, this makes sense to me. Some cases/contexts might not have a result on the contender run but they do have one on the baseline run.